### ISSUE TITLE
No component 'trf_tok2vec' found in pipeline. 

### ISSUE THREAD

**Issue by erotavlas** [explosion/spacy-transformers#147]
I was just wondering if this supports NER?  I noticed some earlier posts (https://github.com/explosion/spacy-transformers/issues/23) with some work in progress and I don't see any examples for NER.  I guess its not released yet, I was getting this error when I tried 

```
Traceback (most recent call last):
  File "ner_kfoldcv.py", line 245, in <module>
    plac.call(main)
  File "D:\Anaconda\envs\transformers\lib\site-packages\plac_core.py", line 367, in call
    cmd, result = parser.consume(arglist)
  File "D:\Anaconda\envs\transformers\lib\site-packages\plac_core.py", line 232, in consume
    return cmd, self.func(*(args + varargs + extraopts), **kwargs)
  File "ner_kfoldcv.py", line 84, in main
    results = pool.starmap(ner_train.worker, args)
  File "D:\Anaconda\envs\transformers\lib\multiprocessing\pool.py", line 274, in starmap
    return self._map_async(func, iterable, starmapstar, chunksize).get()
  File "D:\Anaconda\envs\transformers\lib\multiprocessing\pool.py", line 644, in get
    raise self._value
KeyError: "[E001] No component 'trf_tok2vec' found in pipeline. Available names: ['ner']"

```

I think its from this part of my script

``` 
                …
                ...
		if "ner" not in nlp.pipe_names:
			ner = nlp.create_pipe("ner")
			nlp.add_pipe(ner, last=True)
			reset_weights = True # model contain no pretrained weights for ner
		else: # otherwise, get it so we can add labels
			ner = nlp.get_pipe("ner")

		# add labels
		for _, annotations in TRAIN_DATA:
			for ent in annotations.get("entities"):
				ner.add_label(ent[2])

		# get names of other pipes to disable them during training
		other_pipes = [pipe for pipe in nlp.pipe_names if pipe != "ner"]
		with nlp.disable_pipes(*other_pipes):  # only train NER
			# reset and initialize the weights randomly – but only if we're
			# training a new model
			if model is None or reset_weights:
				nlp.begin_training()
			for itn in range(n_iter):
				random.shuffle(TRAIN_DATA)
				losses = {}
                                …
                                ...
```

**Comment by adrianeboyd** [comment]
NER isn't supported yet, sorry!

**Comment by erotavlas** [comment]
@adrianeboyd  But, this blog post https://explosion.ai/blog/spacy-v2-1

In the post it says it was used already for entity recognizer

> "Even with a small amount of pretraining using default settings, the spacy pretrain command resulted in much better performance for all three components, the tagger, parser and entity recognizer."

**Comment by adrianeboyd** [comment]
Please try out `spacy-nightly` as a preview of how this is implemented in `spacy` v3.0 and `spacy-transformers` v1.0: https://nightly.spacy.io
